frontend/src/metabase/components/DebouncedFrame.jsx

```diff
@@ -45,11 +45,11 @@ export default class DebouncedFrame extends React.Component {
       return;
     }
     if (
-      this.props.width !== nextProps.width ||
-      this.props.height !== nextProps.height
+      this.state.width !== nextProps.width ||
+      this.state.height !== nextProps.height
     ) {
       if (this.state.width == null || this.state.height == null) {
-        this.setSize(nextProps.width, nextProps.height);
+        this.setSizeDebounced(nextProps.width, nextProps.height);
```

Comment:
When we go to the table view, the first time the table is rendered, there wouldn't be a footer, so the table would take the whole space, when when `result` prop changed, the footer will appear causing the table size to be changed, so `setSize` and `setSizeDebounced` will be called in rapid succession causing the transition to appear. I'm not sure why we need to use `setSize` in the first place, but using the debounced version fix that problem.

![image](https://user-images.githubusercontent.com/1937582/157827865-022136b6-852e-4e7b-84f8-405b4179dd17.png)

This wasn't a problem before because `props` and `state` are out of sync due to ``